src/wallet/transaction_builder.rs

```diff
@@ -99,9 +121,10 @@ pub struct TransactionBuilder {
   inputs: Vec<OutPoint>,
   inscriptions: BTreeMap<SatPoint, Vec<InscriptionId>>,
   locked_utxos: BTreeSet<OutPoint>,
+  network: Network,
   outgoing: SatPoint,
-  outputs: Vec<(Address, Amount)>,
-  recipient: Address,
+  outputs: Vec<(OutputType, Amount)>,
```

Comment:
```suggestion
  outputs: Vec<TxOut>,
```

Let's change this to a vec of TxOut since that struct encapsulates these two things.

Reply:
Refactored ✅